salto-io/salto
# SALTO-3984 App Logo, brand Logo and Fav Icon

1. In this PR I added fetch and deploy for app logos, brand logo and favorite icon.
2. Added test for all the filters. 

---

# _Additional context for reviewer_

### To test the code manually do as follows:
1. Fetch Okta env to get all instances.
2. Check that there are static files for appLogos, brandLogos and FavoriteIcon. 
3. change the path for each object type and deploy. 

### Key notes
1. app logo deploy doesn't support removal currently since there isn't API for that and it needs to return to a reset logo (Talked with Shir on it F2F). 
2. Need to add rate limit (Will do after I talk with Neta for the appropriate way to do so).


### Deploy:
Modify: 
change the source file name of appLogo, deploy and check if the appLogo changed in the website. 
`content = file("okta/AppLogo/${oldName}.png")` -> `content = file("okta/AppLogo/{newName.png")`
Add: 
Add application, appLogo and logo static file to your env and then deploy.
you can also add application and app logo and use a logo of a different app.  

---
_Release Notes_: 
_Okta adapter_:
Added option to fetch and deploy app logos, brand theme logo and brand theme icon. 

---
_User Notifications_: 
_Okta adapter_:
Added option to fetch and deploy app logos, brand theme logo and brand theme icon.
